addon/components/osf-copyright/component.js

```diff
@@ -11,5 +11,9 @@ import layout from './template';
  * @class osf-copyright
  */
 export default Ember.Component.extend({
-    layout
+    layout,
+    currentYear: function() {
+        let date = new Date();
```

Comment:
Unit testing tip: for the most part, this will be a straightforward test to write.

Our addons use a tool called Sinon.js to "stub" out things whose behavior we want to control- eg we wouldn't want to have to update the date unit test every year!

These links might be helpful (there's lots of stackoverflow posts of good info also):
http://sinonjs.org/docs/#clock
https://github.com/csantero/ember-sinon

My advice would be to pick a year you find interesting and make sure the component renders that year correctly. Bonus points if I learn something new about that year, but not required. ;)